src/io/flutter/jxbrowser/EmbeddedBrowser.java

```diff
@@ -71,7 +70,7 @@ private EmbeddedBrowser(Project project) {
       // Skip using a transparent background if an exception is thrown.
     } catch (Exception | Error ex) {
       LOG.info(ex);
-      FlutterInitializer.getAnalytics().sendException(StringUtil.getThrowableText(ex), false);
+      FlutterInitializer.getAnalytics().sendCaughtException("embedded-browser-setup: " + ex);
```

Comment:
Here - and for the other jxbrowser related ones - you might instead send an event like:

`sendEvent(JxBrowserManager.ANALYTICS_CATEGORY, "setup:" + ex.getClass().getName());`

Reply:
I wanted to mark all of these as `caught-exception`s, but would that perhaps be better as a label? e.g. eventCategory: `JxBrowserManager.ANALYTICS_CATEGORY`, eventAction: `setup:<Some Exception class>`, eventLabel: `caught-exception` ?

Reply:
If we think there will be expected exceptions in more than just the jxbrowser than having an event category for expected exceptions makes sense. The location / message might then be something like `jsbrowser-setup`?